TestAssets/TestProjects/MSBuildTestApp/MSBuildTestApp.csproj

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Intent here was only just to have something which is working, so we have a stake in the ground to start converting the cli build to msbuild. 

Is it really necessary to work out the final form of everything here before getting something checked in we can iterate on?